tardis/plasma/properties/continuum_processes.py

```diff
@@ -1014,3 +1016,102 @@ def _calculate_factor(self, nu_i, t_electrons):
     def _calculate_u0s(nu, t_electrons):
         u0s = nu[np.newaxis].T / t_electrons * (H / K_B)
         return u0s
+
+
+class CollRecombRateCoeff(ProcessingPlasmaProperty):
+    """
+    Attributes
+    ----------
+    coll_recomb_coeff : pandas.DataFrame, dtype float
+        The rate coefficient for collisional recombination.
+        Multiply with the electron density squared and the ion number density
+        to obtain the total rate.
+
+    Notes
+    -----
+    The collisional recombination rate coefficient is calculated from the
+    collisional ionization rate coefficient based on the requirement of detailed
+    balance.
+    """
+
+    outputs = ("coll_recomb_coeff",)
+    latex_name = (r"c_{\kappa\textrm{i,}}",)
+
+    def calculate(self, phi_ik, coll_ion_coeff):
+        return coll_ion_coeff.multiply(phi_ik.loc[coll_ion_coeff.index])
+
+
+class RawCollIonTransProbs(TransitionProbabilitiesProperty, IndexSetterMixin):
+    """
+    Attributes
+    ----------
+    p_coll_ion : pandas.DataFrame, dtype float
+        The unnormalized transition probabilities for
+        collisional ionization.
+    p_coll_recomb : pandas.DataFrame, dtype float
+        The unnormalized transition probabilities for
+        collisional recombination.
+    C_coll_ion : pandas.DataFrame, dtype float
```

Comment:
Can this variable have a more descriptive name such as `coll_ion_cool_rate` or similar?